pkg/cluster/ClusterService.go

```diff
@@ -493,10 +494,10 @@ func (impl *ClusterServiceImpl) Update(ctx context.Context, bean *ClusterBean, u
 	model.PrometheusEndpoint = bean.PrometheusUrl
 
 	if bean.PrometheusAuth != nil {
-		if bean.PrometheusAuth.UserName != "" {
+		if bean.PrometheusAuth.UserName != "" || bean.PrometheusAuth.IsAnonymous {
```

Comment:
what if bean.PrometheusAuth.UserName=="" while IsAnonymous, will this work with empty username

Reply:
yes , when we have empty username it is a case of anonymous type ,I have tested for these cases.